VisualRiders.PointOfSale.Project/DTOs/ReadTaxDto.cs

```diff
@@ -0,0 +1,23 @@
+using System.ComponentModel.DataAnnotations;
+using VisualRiders.PointOfSale.Project.Models;
+
+namespace VisualRiders.PointOfSale.Project.DTOs;
+
+public class ReadTaxDto
+{
+    [Required]
```

Comment:
Data annotations are not needed for read DTOs as there is no validation hapenning here. These annotation are only used to prevent clients from passing invalid data.